source/androidx.media3/media3-common/Transforms/Metadata.xml

```diff
@@ -0,0 +1,161 @@
+        name="managedName"
+        >
+        OnCuesDeprecated
+    </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.common']/interface[@name='Player.Listener']/method[@name='onPositionDiscontinuity' and count(parameter)=1 and parameter[1][@type='int']]"
+        name="managedName"
+        >
+        OnPositionDiscontinuityDeprecated
+    </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.common']/class[@name='DrmInitData']/method[@name='compare' and count(parameter)=2 and parameter[1][@type='androidx.media3.common.DrmInitData.SchemeData'] and parameter[2][@type='androidx.media3.common.DrmInitData.SchemeData']]/parameter[1]"
+        name="managedType"
+        >
+        Java.Lang.Object
+    </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.common']/class[@name='DrmInitData']/method[@name='compare' and count(parameter)=2 and parameter[1][@type='androidx.media3.common.DrmInitData.SchemeData'] and parameter[2][@type='androidx.media3.common.DrmInitData.SchemeData']]/parameter[2]"
+        name="managedType"
+        >
+        Java.Lang.Object
+    </attr>
+    <remove-node
+        path="/api/package[@name='androidx.media3.common']/interface[@name='SimpleBasePlayer.PositionSupplier']/field[@name='ZERO']"
+        />
+    <remove-node
+        path="/api/package[@name='androidx.media3.common']/interface[@name='SimpleBasePlayer.PositionSupplier']/method[@name='getConstant' and count(parameter)=1 and parameter[1][@type='long']]"
+        />
+    <remove-node
+        path="/api/package[@name='androidx.media3.common']/interface[@name='SimpleBasePlayer.PositionSupplier']/method[@name='getExtrapolating' and count(parameter)=2 and parameter[1][@type='long'] and parameter[2][@type='float']]"
+        />
+    <attr
+        path="/api/package[@name='androidx.media3.common']/interface[@name='Player']/method[@name='addMediaItems' and count(parameter)=2 and parameter[1][@type='int'] and parameter[2][@type='java.util.List&lt;androidx.media3.common.MediaItem&gt;']]/parameter[2]"
+        name="managedType"
+        >
+        System.Collections.Generic.IList &lt; AndroidX.Media3.Common.MediaItem &gt;
+    </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.common']/interface[@name='Player']/method[@name='addMediaItems' and count(parameter)=1 and parameter[1][@type='java.util.List&lt;androidx.media3.common.MediaItem&gt;']]/parameter[1]"
+        name="managedType"
+        >
+        System.Collections.Generic.IList &lt; AndroidX.Media3.Common.MediaItem &gt;
+    </attr>
+    <attr
+        path="/api/package[@name='androidx.media3.common']/interface[@name='Player']/method[@name='addMediaItems' and count(parameter)=2 and parameter[1][@type='int'] and parameter[2][@type='java.util.List&lt;androidx.media3.common.MediaItem&gt;']]"
+        name="abstract"
```

Comment:
Marking this a `!abstract` will mean the user will not be required to provide an implementation and any calls to this method will fail.